src/mandelbrot.rs

```diff
@@ -0,0 +1,104 @@
+// The Computer Language Benchmarks Game
+// http://benchmarksgame.alioth.debian.org/
+//
+// contributed by Matt Watson
+// contributed by TeXitoi
+
+use std::io::Write;
+use std::thread;
+const THREADS: usize = 8;
+const MAX_ITER: usize = 50;
+const VLEN: usize = 8;
+const ZEROS: Vecf64 = [0.0; VLEN];
+
+pub type Vecf64 = [f64; VLEN];
+
+fn mul2 (x: Vecf64, y: Vecf64) -> Vecf64 {
+    let mut res = ZEROS;
+    for i in 0..VLEN {res[i] = x[i] * y[i];}
+    res
+}
+fn add2 (x: Vecf64, y: Vecf64) -> Vecf64 {
+    let mut res = ZEROS;
+    for i in 0..VLEN {res[i] = x[i] + y[i];}
+    res
+}
+fn sub2 (x: Vecf64, y: Vecf64) -> Vecf64 {
+    let mut res = ZEROS;
+    for i in 0..VLEN { res[i] = x[i] - y[i]; }
+    res
+}
+
+pub fn mbrot8(cr: Vecf64, ci: Vecf64) -> u8 {
```

Comment:
Yeah. I didn't follow exactly what it was doing, but it seemed to run out of registers and resort to accessing the cache much sooner.

http://goo.gl/nhpT4f

Looking at it more closely, it isn't using any ymm registers and is directly substituting avx instructions for SSE3 ones...odd. It's irrelevant to the game though and is the kind of thing that will probably turn up in later versions of LLVM as the instructions get older.